src/components/LogInPage.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate, Link } from 'react-router-dom';
+import { login } from '../redux/user/userAPI';
+import '../styles/LogInPage.css';
+
+const CriptoJS = require('crypto-js');
+
+const Login = () => {
+  const user = useSelector((store) => store.user);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const [value, setValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  const onChange = (e) => {
+    setValue({
+      ...value,
+      [e.target.id]: e.target.value,
+    });
+    console.log(value);
```

Comment:
- [ ]  Please, kindly remove this old piece of debugging code from the repo (console.log).  That way, it will be cleaner.  💪